### PR TITLE
Use uppercase keywords

### PR DESCRIPTION
**Pull request by szarnyasg**

### CODE REVIEW

File: docs/sql/configuration.md [hunk 1]
```diff
@@ -35,70 +35,70 @@ Below is a list of all available settings.
 
 |                   name                   |                                                                       description                                                                       | input_type |      default_value       |
 |------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|------------|--------------------------|
-| Calendar                                 | The current calendar                                                                                                                                    | VARCHAR    | System (locale) calendar |
```
Comment (Mause) [inline docs/sql/configuration.md:38]: Please update the generator for this change

**Review by Mause — CHANGES_REQUESTED** [review]

File: docs/sql/configuration.md (hunk 1 above)
Comment (szarnyasg) [reply docs/sql/configuration.md]: Good point, thx. I dropped this last commit from the PR.

**Comment by szarnyasg** [comment]
The rest of the PR is trivial, so lmk if it's good to go.

**Comment by Mause** [comment]
> The rest of the PR is trivial, so lmk if it's good to go.

Better to update the generator instead and remove your changes here, else you'll end up with painful merge conflicts

**Comment by szarnyasg** [comment]
Yes, I force-pushed my last commit (that targeted `configuration.md`) out of this PR to avoid any potential merge conflicts.

**Review by Mause — APPROVED** [review]

**Pull request merged**